tools/cli/pkg/printer/table.go

```diff
@@ -234,19 +234,28 @@ func (t *tablePrinter) printOneObj(obj interface{}) error {
 			lastOp := *eventList[0].OperationID
 			buffer := strings.Builder{}
 			eventTabWriter := newTabWriter(&buffer)
-			printOperation(eventTabWriter, lastOp)
+			rt := obj.(runtime.RuntimeDTO)
```

Comment:
technically `rt` is equal to `r`, but the `RuntimeDTO` doesn't contain info about all `operations` so we will need to figure out a different way.